web/src/components/selection.json

```diff
@@ -1,6 +1,52 @@
 {
   "IcoMoonType": "selection",
   "icons": [
+    {
```

Comment:
how did you generate this?

Reply:
https://wiki.replicated.io/doc/using-and-updating-icons-in-admin-console-yE7HD4tTrw